addons/mail/static/tests/crosstab/crosstab.test.js

```diff
@@ -41,10 +41,12 @@ test("Delete starred message updates counter", async () => {
         res_id: channelId,
         starred_partner_ids: [serverState.partnerId],
     });
+    onRpc("/discuss/channel/messages", () => step("/discuss/channel/messages"));
     const env1 = await start({ asTab: true });
     const env2 = await start({ asTab: true });
     await openDiscuss(channelId, { target: env1 });
     await openDiscuss(channelId, { target: env2 });
+    await assertSteps(["/discuss/channel/messages", "/discuss/channel/messages"]);
     await contains("button", { target: env2, text: "Starred1" });
     rpc("/mail/message/update_content", {
```

Comment:
Why do we open 2 tabs in this test if we're making the rpc manually anyway rather than from the other tab?

Reply:
It would be better to do so. However the test still has value since it ensures the starred counter updates in the other tab.